### PR TITLE
Make draggable CSS property customisable

### PR DESCRIPTION
**Pull request by leaanthony**
Adds 2 new applications allowing the developer to choose which CSS property and value should be used to indicate drag regions.

### CODE REVIEW

File: website/docs/reference/options.mdx [hunk 1]
```diff
@@ -40,6 +40,8 @@ func main() {
         OnShutdown:         app.shutdown,
         OnBeforeClose:      app.beforeClose,
         WindowStartState:   options.Maximised,
+        CSSDragProperty:   "widows",
```
Comment (stffabi) [inline website/docs/reference/options.mdx:43]: Should we use the default value here?
```suggestion
        CSSDragProperty:   "--wails-draggable",
```

File: website/docs/reference/options.mdx [hunk 2]
```diff
@@ -40,6 +40,8 @@ func main() {
...
+        CSSDragValue:      "1",
```
Comment (stffabi) [inline website/docs/reference/options.mdx:44]: Should we use the default value here?

```suggestion
        CSSDragValue:      "drag",
```

**Review by stffabi — APPROVED** [review]
LGTM, just two minor comments about the docs.

**Comment by leaanthony** [comment]
Thanks @stffabi 🙏

**Pull request merged**